src/sampler.jl

```diff
@@ -15,7 +15,7 @@ function sample_init(
 end
 
 # A step is a momentum refreshment plus a transition
-function step(
+function transit(
```

Comment:
Here it does not seem to add anything by changing the function name.  I would keep the old function name if there are no strong reasons. The function name `step` is more intuitive than `transit` -- `transit` sounds a bit mysterious, I think. Plus, this is a breaking change, although currently only depreciated.

Reply:
Sure. I can revert this.

Reply:
OK so I reverted it. But again it would be nice to have a seperate name so that `step` for leapfrog is namly different from `step` for MCMC transition.